runhouse/main.py

```diff
@@ -452,33 +453,48 @@ def status(
             )
             raise typer.Exit(1)
 
+    if cluster_or_local == "file" and not cluster_name:
+        # If running outside the cluster must specify a cluster name
+        console.print("Missing argument `cluster_name`.")
+        return
+
     # case we are inside the cluster
-    if cluster_or_local != "file":
+    elif cluster_or_local != "file":
+
         # If we are on the cluster load status directly from the object store
-        cluster_status: dict = dict(obj_store.status())
-        cluster_config = copy.deepcopy(cluster_status.get("cluster_config"))
+        # cluster status type: ResourceStatusData
+        cluster_status, send_status_resp = obj_store.status(send_to_den=True)
```

Comment:
i think we can consolidate this logic, and have everything flow thru: 
```
current_cluster.status(
                resource_address=current_cluster.rns_address, send_to_den=True
            )
```

since this will check if we are on the cluster, and call the object store if so anyway